<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Searchmongo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_and">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>AND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_or">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_query">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_create_date">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:CreateDate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sourcefile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SourceFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_filename">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>File:FileName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_directory">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>File:Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_make">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:Make</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_model">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyright">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:Copyright</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exposuretime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:ExposureTime</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iso">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:ISO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aperture_value">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:ApertureValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_focal_lenght">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:FocalLength</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_focal_lenght_35">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:FocalLengthIn35mmFormat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lens_model">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIF:LensModel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eq">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Equal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Greater than</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gte">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Greater than equall</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Less than</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lte">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Less than equal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ne">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Not equal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Values</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_query">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>1011</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_labels">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>IPCT:Keywords</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_values">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_end_query">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>750</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>End Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_query">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>750</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hlp">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Help Box</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_all">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>labels all</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rb_or">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>lbels or</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="le_labels">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label list withou quotes and separated by comma</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_or">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear_query</sender>
   <signal>clicked()</signal>
   <receiver>lbl_query</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>925</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
